examples/src/keyrings/set_encryption_algorithm_example.py

```diff
@@ -0,0 +1,144 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""
+This example demonstrates how to set an encryption algorithm while using the Raw AES Keyring
+in the AWS Encryption SDK.
+
+Encryption algorithms can be set in a similar manner in other keyrings as well. However,
```

Comment:
```suggestion
in the AWS Encryption SDK.

The encryption algorithm used in the encrypt() method is the
algorithm used to protect your data using the data key.
By setting this algorithm, you can configure the algorithm
used to encrypt and decrypt your data.

Encryption algorithms can be set in a similar manner in other keyrings as well. However,
```

I want a sentence as a subheader that really briefly explains the "what" and "why"